lib/utils/SOAPHeader.js

```diff
@@ -96,9 +97,10 @@ const PASSWORD_TEXT = 'http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-u
    * Return the object for username token to parse
    * @param {SamlTokenInput} samlTokenInput token information
    * @param {object} jObj object to parse
+   * @param {string} parserAttributePlaceHolder character for attributes of parser
    * @returns {object} jObj object to parse
    */
-  processSAMLToken = (samlTokenInput, jObj) => {
+  processSAMLToken = (samlTokenInput, jObj, parserAttributePlaceHolder) => {
```

Comment:
@LuisTejedaS This seems to be the wrong definition of the function of class. it should be as following.

```
processSAMLToken(samlTokenInput, jObj, parserAttributePlaceHolder) {
```